jujugui/static/gui/src/test/test_env_api.js

```diff
@@ -559,6 +559,37 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
     });
 
+    describe('redirectInfo', function() {
+      it('makes appropriate request to fetch model redirect info', function() {
+        env.redirectInfo();
+        var lastMessage = conn.last_message();
+        var expected = {
+          params: {},
+          request: 'RedirectInfo',
+          'request-id': 1,
+          type: 'Admin',
+          version: 3
+        };
+        assert.deepEqual(expected, lastMessage);
+      });
+
+      it('responds with only the server data if no error', function() {
+        const callback = sinon.stub();
+        env.redirectInfo(callback);
+        conn.msg({ 'request-id': 1, response: { servers: 'servers' }});
```

Comment:
I'd use `{servers: [{...}, {...}]}` to make the mock more realistic and the test more informative for future readers.